preamble.tex

```diff
@@ -101,6 +101,10 @@
 \end{nonnormative}
 }
 
+% Semantical markup with formatting
+\newcommand{\firstuse}[1]{\emph{#1}}
```

Comment:
The name of the command  is confusing. Unless you want to couple this with some glossary function i don't see why you would call this ''first''use.

Reply:
Maybe go the whole way now and start using [glossaries](https://en.wikibooks.org/wiki/LaTeX/Glossary)

Reply:
Yeah, I actually had coupling with glossary in mind.  I really like the idea of using something like the _glossaries_ package, but in case that is too much to include in this PR, the current use of `\firstuse` would be a good starting point.

Reply:
> 
> 
> Maybe go the whole way now and start using [glossaries](https://en.wikibooks.org/wiki/LaTeX/Glossary)

Interesting, but to me a glossary would be a bit odd, at least for the terms that the specification defines (like stream variable). However, using it as basis for an index seems similar, and straightforward (because then we get a link back to everything, including examples).